dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/importer/EventManager.java

```diff
@@ -293,6 +324,57 @@ public ImportSummaries deleteEvents( final List<Event> events, final WorkContext
         return importSummaries;
     }
 
+    private void logTrackedEntityDataValueHistory( Event event, WorkContext workContext, Date today )
```

Comment:
not sure about our naming conventions when it comes to auditing. I would have expected us to literally log due to the method name. Since we don't log, I would name it `auditTrackedEntityDataValueHistory`. Using the same language you used in the jira task.